app/controllers/api/v3/scenarios_controller.rb

```diff
@@ -88,7 +88,7 @@ def create
 
         @scenario = Scenario.new
 
-        if scaler_attributes && ! attrs[:descale]
+        if scaler_attributes && ! attrs[:descale] && ! Area.derived?(attrs.fetch(:area_code))
```

Comment:
I think that this code will not correctly work with the idea that a scenario from a derived dataset can scale. Mainly because of [these lines](https://github.com/quintel/etengine/blob/local-datasets-scaling-of-sliders/app/models/scenario_scaling.rb#L27-L33). 

I'll try to spec this though and see if that'll work. But it's good to know that descaling will descale back to the `derived_dataset` area. 👍